src/object/obj_tx.c

```diff
@@ -2787,6 +2809,9 @@ dc_tx_per_akey_existence_task(enum obj_rpc_opc opc, daos_handle_t oh, struct dc_
 			D_MUTEX_UNLOCK(&tx->tx_lock);
 			obj_decref(obj);
 
+			if (tx->tx_for_convert)
+				rc = dc_tx_convert_post(tx, parent, opc, 0);
+
 			/* Drop the reference that is held via dc_tx_attach(). */
 			dc_tx_decref(tx);
 		} else {
```

Comment:
[Question] In this "else" case, where do we call dc_tx_convert_post?

Reply:
I see. I didn't find that last time...

Reply:
Not your issue. I should checked more carefully during last merge with master.